src/callbacks_step/analysis_dg2d.jl

```diff
@@ -236,6 +236,20 @@ function analyze(::Val{:l2_divb}, du, u, t,
   end |> sqrt
 end
 
+function analyze(::Val{:l2_divb}, du, u, t,
```

Comment:
I think adding a second `l2_divb` function is ok, but once we add a third version because the indices of B are different, we should consider a refactoring into a single `analyze_divb` that takes the relevant indices as arguments.

Reply:
I removed the specialized divB routines for multi-ion MHD in a555e558 after unifying them in https://github.com/trixi-framework/Trixi.jl/pull/2176.

Reply:
Great, thanks!